dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/mapper/AccessTokenMapperTest.java

```diff
@@ -77,14 +76,7 @@ public void testInsert() throws Exception{
     @Test
     public void testSelectById() throws Exception{
         Integer userId = 1;
-
-        AccessToken accessToken = createAccessToken(userId);
-
-        AccessToken resultAccessToken
-                = accessTokenMapper.selectById(accessToken.getId());
```

Comment:
why remove this

Reply:
this accessToken different resultAccessToken in memory address，so use assertEquals(accessToken，resultAccessToken) are error in compile time

Reply:
you are right. but we can optimize rather than delete them.